exe/dirtree

```diff
@@ -84,6 +93,18 @@ end
 
 if options.key?(:output)
   File.write(options[:output], result)
+elsif options.key?(:screenshot)
+  tmp = Tempfile.new(default_file_name)
+  tmp.write(result)
+  tmp.close
 else
   puts result
 end
+
+if options[:screenshot]
+  file = options[:output] || tmp.path
+  system('chromium', '--disable-gpu', '--headless', '--screenshot', file) ||
+    system('chrome', '--disable-gpu', '--headless', '--screenshot', file)
```

Comment:
I didn't know that `chrome` has that feature 😄 
but I think this needs to have more fall backs here, 
as in my archlinux setup for example google chrome binary is named `google-chrome` and another binary is named `google-chrome-stable` 
and if all fails we can show a message of failure

and to implement that we can represent the possible names for google chrome in a list then trying them one by one until we find a successful candidate.